docker-compose/technical/create-cassandra-databases.sh

```diff
@@ -3,12 +3,8 @@
 set -e
 
 cat > create_keyspaces.cql <<EOF
-CREATE KEYSPACE IF NOT EXISTS "${DATABASE_PREFIX_NAME}cgmes_assembling" WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1 };
 EOF
 
-echo "USE ${DATABASE_PREFIX_NAME}cgmes_assembling;" >> init_keyspaces.cql
-curl https://raw.githubusercontent.com/gridsuite/cgmes-assembling-job/main/src/main/resources/cgmes_assembling.cql >> init_keyspaces.cql
-
 until cqlsh -f /create_keyspaces.cql; do
   echo "cqlsh: Cassandra is unavailable to create keyspaces - will retry later"
   sleep 2
```

Comment:
init_keyspaces.cql doesn't exist anymore.
Ok then remove everything in this file except the last command
`exec /docker-entrypoint.sh "$@"`